hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/job/PinotBuildAndPushJob.groovy

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.gradle.hadoopdsl.job
```

Comment:
@jenniferdai Can you remove the extra line between the license disclaimer and the package declaration, and add comments to the package declaration and import statements (this convention matches the rest of the files in the project).